docs/manage/troubleshoot/troubleshoot.md

```diff
@@ -19,6 +19,95 @@ For common errors see [Common Errors](/dev/tools/common-errors/).
 
 ## Check logs
 
+### Machine shows as offline
+
+If your machine shows as offline in the Viam app, restart `viam-server` by running the command to start `viam-server` and adding the `-debug` option.
+
+To do this, you will need to know if you installed `viam-server` with `viam-agent` (most common) or manually.
```

Comment:
There are mac instructions below. Do you want to also clarify up here that this doesn't apply for mac users, since agent doesn't run on mac?

Reply:
Mac users can run these instructions too if they don't immediately know.